docs/redis-clients/python.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Python guide"
+linkTitle: "Python"
+description: Connect your Python application to a Redis database
+weight: 1
+
+---
+
+Use one of the following options to connect your application to a Redis database:
```

Comment:
It's not related to this documentation PR. Nevertheless, we are missing some additional doc site features like tabs. Tabs like on Stripe documentation make perfect sense for this kind of list:

https://stripe.com/docs/development/quickstart
<img width="882" alt="image" src="https://user-images.githubusercontent.com/1655867/208058229-d0fe224b-f130-4235-9754-d5aabaaae33b.png">





Also, It will get worse when we add sections for specific mainstream frameworks like FastAPI.

Reply:
[](url)

Reply:
[](url)